<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotDeletedDlg</class>
 <widget class="QDialog" name="PlotDeletedDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deleted documents</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <widget class="QStackedWidget" name="swTop">
     <widget class="QWidget" name="pgList">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Start with</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbStartWith">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Show count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbShowCount">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbPrev">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbNext">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pbShow">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="PlotListTree" name="twDocs">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotListTree</class>
   <extends>QTreeWidget</extends>
   <header>../VProject/PlotListTree.h</header>
   <slots>
    <signal>AskMakeNewDocument()</signal>
    <signal>WasUndeleted()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>twDocs</sender>
   <signal>WasUndeleted()</signal>
   <receiver>PlotDeletedDlg</receiver>
   <slot>RequeryData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbShow</sender>
   <signal>clicked()</signal>
   <receiver>PlotDeletedDlg</receiver>
   <slot>ShowData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ShowData()</slot>
  <slot>RequeryData()</slot>
 </slots>
</ui>
